pkg/build/golang.go

```diff
@@ -18,14 +15,14 @@ func golang(root string, args Args) (string, error) {
 	var entrypoint = args["entrypoint"]
 	var main = filepath.Join(root, entrypoint)
 
-	if err := exist(gomod, gosum, main); err != nil {
+	if err := exist(gomod, main); err != nil {
 		return "", err
 	}
 
 	t, err := template.New("golang").Parse(`
 FROM golang:1.16.0-alpine3.13 as builder
 WORKDIR /airplane
-COPY go.mod go.sum ./
+COPY go.mod {{ if .HasGoSum -}} go.sum {{ end -}} ./
```

Comment:
lol minor but maybe just avoid the `-` business

```suggestion
COPY go.mod {{ if .HasGoSum }}go.sum {{ end }}./
```